fvm/blueprints/system.go

```diff
@@ -31,11 +26,6 @@ func prepareSystemContractCode(chainID flow.ChainID) string {
 		systemChunkTransactionTemplate,
 		sc.AsTemplateEnv(),
 	)
-	code = strings.ReplaceAll(
-		code,
-		placeholderEVMAddress,
-		sc.EVMContract.Address.HexWithPrefix(),
-	)
 	code = strings.ReplaceAll(
```

Comment:
Should this be removed too?

Reply:
`Migration` isn't in `flow-core-contracts` yet, so I didn't really want to change anything about it here until we add it there